code/src/reducers/toread.js

```diff
@@ -0,0 +1,67 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const toread = createSlice({
+  name: "toread",
+  initialState: {
+    books: [
+      {
+        id: 1,
+        title: "Nocturner",
+        isRead: true,
+      },
+      {
+        id: 2,
+        title: "The white book",
+        isRead: true,
+      },
+      {
+        id: 3,
+        title: "The Great Gatsby",
+        isRead: false,
+      },
+      {
+        id: 4,
+        title: "Swede Hollow",
+        isRead: true,
+      },
+      {
+        id: 5,
+        title: "In the fold",
+        isRead: false,
+      },
+    ],
+  },
+  reducers: {
+    addBook: (store, action) => {
+      const newBook = {
+        id: Math.max(...store.books.map((book) => book.id)) + 1,
```

Comment:
With this logic, what happens if the array is empty? Look into another way of creating the id, e.g. https://www.npmjs.com/package/uuid